### ISSUE TITLE
fatal: 'gcc-8_3_0-release' is not a commit and a branch '8' cannot be created from it

### ISSUE THREAD

**Issue by lvivtotoro** [sba1/adtools#81]
I was simply following the steps in the README and when I got to `bin/gild checkout gcc 8`, I got this output:

```
fatal: 'gcc-8_3_0-release' is not a commit and a branch '8' cannot be created from it
fatal: 'gcc-8_3_0-release' is not a commit and a branch '8' cannot be created from it
Failed to checkout 8
``` Same error occurs with versions 5 and 6. All previous steps seem to have been successful.

**Comment by lvivtotoro** [comment]
I tried again today and now I get a different error, *again* at the gcc step:

```
Cloning into 'repo'...
warning: Could not find remote branch gcc-8_3_0-release to clone.
fatal: Remote branch gcc-8_3_0-release not found in upstream origin
Traceback (most recent call last):
  File "/opt/adtools/gild/bin/gild-checkout", line 68, in <module>
    os.chdir(repo_path)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/adtools/gcc/repo'
```

**Comment by cpm1** [comment]
Workaround is to change `gcc/series` file, last line to:
`8       releases/gcc-8.3.0      http://ftp.gnu.org/gnu/gcc/gcc-8.3.0/gcc-8.3.0.tar.xz`

**Comment by kas1e** [comment]
Yeah, I can confirm that the issue still there.  But workaround from cpm1 works as well

**Comment by sba1** [comment]
Fixed with aecbbd581ae4b293d5af75089c1813bf17301f01.
